numba_dpex/dpnp_iface/_intrinsic.py

```diff
@@ -960,10 +1008,17 @@ def impl_dpnp_full_like(
         ty_sycl_queue,
         ty_retty_ref,
     )
+    sycl_queue_arg_pos = -2
+    array_arg_pos = 0
 
     def codegen(context, builder, sig, args):
         qref_payload: _QueueRefPayload = _get_queue_ref(
-            context, builder, sig, args
+            context,
+            builder,
+            sig,
+            args,
+            sycl_queue_arg_pos=sycl_queue_arg_pos,
```

Comment:
Generally, I will like to avoid global variables. I also wish to find a way to not have to do the hard coded variables, just could not find a way out of properly using keyword args.